src/main.cpp

```diff
@@ -4201,14 +4210,17 @@ bool CVerifyDB::VerifyDB(CCoinsView *coinsview, int nCheckLevel, int nCheckDepth
         // check level 3: check for inconsistencies during memory-only disconnect of tip blocks
         if (nCheckLevel >= 3 && pindex == pindexState && (coins.DynamicMemoryUsage() + pcoinsTip->DynamicMemoryUsage()) <= nCoinCacheUsage) {
             bool fClean = true;
```

Comment:
Looks good. Only difference I can see is that `bool fClean = true;` is removed in the upstream PR. I double checked and we can safely remove this.

```suggestion
```

Reply:
Thanks, @Eirik0, good catch. This is fixed in PR #3837 (https://github.com/zcash/zcash/pull/3837/files#diff-7ec3c68a81efff79b6ca22ac1f1eabbaL4203), which the current PR is a prelude to, so if you don't mind, let's not fix this as part of the current PR (I can't easily force-push because that other commit is already on top).

Reply:
It's pretty trivial and it won't have any effect on the execution of the code, so if it's easier to do it later I am happy with that.